capistrano3-puma.gemspec

```diff
@@ -10,14 +10,14 @@ Gem::Specification.new do |spec|
   spec.homepage = 'https://github.com/seuros/capistrano-puma'
   spec.license = 'MIT'
 
-  spec.required_ruby_version = '>= 2.5'
+  spec.required_ruby_version = Gem::Requirement.new(">= 3.0")
```

Comment:
This change shouldn't be necessary. Just because Puma 7 dropped support for those older Ruby versions, doesn't mean that this gem should do so, because this gem can still deploy puma versions that support those old Ruby versions.

Reply:
I will fix it  before release, let not block the merge. 

Sorry, i did not notice the notification earlier.